src/main/java/frc/robot/commands/autoCommands/waitCommand.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands.autoCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class waitCommand extends Command {
+
+  private int delay;
+  private int curTimeWaited = 0;
+
+  public waitCommand(int delay) {
+    this.delay = delay;
+  }
+
+  @Override
+  public void initialize() {
+    this.curTimeWaited = 0;
+  }
+
+  @Override
+  public void execute() {
+    this.curTimeWaited++;
+  }
+
+  @Override
+  public void end(boolean interrupted) {}
+
+  @Override
+  public boolean isFinished() {
+    return this.curTimeWaited > this.delay;
```

Comment:
I think this should be >=, but I actually don't want to sit down and write it out. This is more intuition from me, but I'm pretty sure this would also cause an extra 20 ms delay in the loop. initialize gets called on the schedule, execute gets called every loop after that, and isFinished gets called after execute. So I think it would be an extra loop with just >.